src/function/scalar/map/map_entries.cpp

```diff
@@ -0,0 +1,93 @@
+#include "duckdb/planner/expression/bound_function_expression.hpp"
+#include "duckdb/common/string_util.hpp"
+#include "duckdb/parser/expression/bound_expression.hpp"
+#include "duckdb/function/scalar/nested_functions.hpp"
+#include "duckdb/common/types/data_chunk.hpp"
+#include "duckdb/common/pair.hpp"
+
+namespace duckdb {
```

Comment:
Mostly the same comments as for `map_keys.cpp`. Try to reduce the amount of duplicated code. :)